### PR TITLE
Add `Corgi Cup: June 2016` tournament article

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Adds the full Corgi Cup: June 2016 tournament article, also fixes a minor error in the Corgi Cup VI article.

- [x] Waiting on a quick check by doomsday.

### CODE REVIEW

**Comment by Bazzadwarf** [comment]
This is ready for a review now!

File: wiki/Tournaments/CC/1/en.md [hunk 1]
```diff
@@ -8,11 +8,237 @@ tags:
...
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
+
+## Organisation
+
+Corgi Cup: June 2016 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_FI] [M8gazine](https://osu.ppy.sh/users/5039710) |
+| Co-Host | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) |
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:41]:
```suggestion
| Co-host | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) |
```

File: wiki/Tournaments/CC/1/en.md [hunk 2]
```diff
@@ -8,11 +8,237 @@ tags:
...
+  - [Shawn Wasabi - Marble Soda (Len) [Crier's Extra]](https://osu.ppy.sh/beatmapsets/387784#osu/846259)
+  - [S-C-U - concon (Sync) [Extreme]](https://osu.ppy.sh/beatmapsets/42051#osu/133053)
+- DoubleTime
+  - [HTT - Pure Pure Heart (miccoliasms) [miccoli]](https://osu.ppy.sh/beatmapsets/16523#osu/67469)
+  - [Sakura Chiyo (CV:Ozawa Ari) - URAOMOTE FORTUNE (P A N) [Insane]](https://osu.ppy.sh/beatmapsets/210706#osu/495590)
+- FreeMod
+  - [Telekinesis - The Monk (FCL) [Technique of Shadow]](https://osu.ppy.sh/beatmapsets/381529#osu/834589)
+  - [IRON ATTACK! - Future is Undefined (Kite) [Insane]](https://osu.ppy.sh/beatmapsets/32447#osu/106156)
+- Tiebreaker
+  - **[SawanoHiroyuki\[nZk\]:Gemie - x.U. (ETIA. Hardcore Bootleg Remix) (captin1) [Shinoa]](https://osu.ppy.sh/beatmapsets/331872#osu/735021)**
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:119]:
```suggestion
  - **[SawanoHiroyuki[nZk]:Gemie - x.U. (ETIA. Hardcore Bootleg Remix) (captin1) [Shinoa]](https://osu.ppy.sh/beatmapsets/331872#osu/735021)**  
```

File: wiki/Tournaments/CC/1/en.md [hunk 3]
```diff
@@ -8,11 +8,237 @@ tags:
...
+  - [IRON ATTACK! - Future is Undefined (Kite) [Insane]](https://osu.ppy.sh/beatmapsets/32447#osu/106156)
+- Tiebreaker
+  - **[SawanoHiroyuki\[nZk\]:Gemie - x.U. (ETIA. Hardcore Bootleg Remix) (captin1) [Shinoa]](https://osu.ppy.sh/beatmapsets/331872#osu/735021)**  
+
+### Quarterfinals
+
+- NoMod
+  - [Nekomata Master - Far east nightbird (kors k Remix) (jonathanlfj) [RLC's Extra]](https://osu.ppy.sh/beatmapsets/144171#osu/358273)
+  - [Traktion - Gid V (Irreversible) [galV's Insane]](https://osu.ppy.sh/beatmapsets/163689#osu/399252)
+  - [Dark PHOENiX - Stirring an Autumn Moon (_lolipop) [Crazy Moon]](https://osu.ppy.sh/beatmapsets/16650#osu/59693)
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:126]:
```suggestion
  - [Dark PHOENiX - Stirring an Autumn Moon (\_lolipop) [Crazy Moon]](https://osu.ppy.sh/beatmapsets/16650#osu/59693)
```

File: wiki/Tournaments/CC/1/en.md [hunk 4]
```diff
@@ -8,11 +8,237 @@ tags:
...
+| **Pelzio** ![][flag_GB] | **4** | 1 | ![][flag_GB] MagicMantwon | [#1](https://osu.ppy.sh/community/matches/25711910) |
+| Lazulite ![][flag_US] | 3 | **4** | ![][flag_GB] **Hydria** | [#1](https://osu.ppy.sh/community/matches/25712820) |
+| **rafalbrew** ![][flag_GB] | **4** | 0 | ![][flag_GB] SteffoST | [#1](https://osu.ppy.sh/community/matches/25713279) |
+| **Seraxis** ![][flag_GB] | **4** | 0 | ![][flag_CZ] NyarkoO | [#1](https://osu.ppy.sh/community/matches/25715231) |
+| **zelda101** ![][flag_GB] | **4** | 1 | ![][flag_DE] Blegie | [#1](https://osu.ppy.sh/community/matches/25715822) |
+| YouKittenMe ![][flag_GB] | 2 | **4** | ![][flag_US] **MicMck101** | [#1](https://osu.ppy.sh/community/matches/25716816) |
+
+## Ruleset
+
+- This tournament is a 1v1, Head to head, Single Elimination tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:224]:
```suggestion
- This is a 1v1, head-to-head, single-elimination tournament.
```

File: wiki/Tournaments/CC/1/en.md [hunk 5]
```diff
@@ -8,11 +8,237 @@ tags:
...
+- Each round will be BO7 (first to 4).
+- Referees create the match and will invite players 10 mins prior to match beginning.
+- If a player fails to turn up before or on the time their match is due to start, a 10 minute timer will be started. After 10 minutes, if the player has still not shown up, the other player wins.
+- If both players fail to show up, the match can be rescheduled during the times given.
+- Warm-ups are optional should a player request one. (Note: the warm up should not be from the mappool the players are due to play and must have a drain time of 4 mins or less.)
+- After warm ups (should there be any), both players will `!roll` to decide who bans/picks first. Loser of the roll bans first and winner of the roll picks first.
+- Each player is allowed one ban per match (Not Tiebreaker).
+- If a player disconnects during the match in which the outcome would have been different. A rematch can be offered under the Referee’s discretion.
+- Hard Rock and Double Time mappools can be played with Hidden should the player want to.
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:233]:
```suggestion
- Hard Rock and Double Time mappools can be played with Hidden if the player wants to.
```

File: wiki/Tournaments/CC/1/en.md [hunk 6]
```diff
@@ -8,11 +8,237 @@ tags:
...
+- Score V2 will be used.
+- Free mod can be any mod the player decides (including EZ + FL).
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:235]:
```suggestion
- Free Mod can be any mod the player decides (including EZ + FL).
```

File: wiki/Tournaments/CC/1/en.md [hunk 7]
```diff
@@ -8,11 +8,237 @@ tags:
...
+## Ruleset
+
+- This tournament is a 1v1, Head to head, Single Elimination tournament.
+- Each round will be BO7 (first to 4).
+- Referees create the match and will invite players 10 mins prior to match beginning.
+- If a player fails to turn up before or on the time their match is due to start, a 10 minute timer will be started. After 10 minutes, if the player has still not shown up, the other player wins.
+- If both players fail to show up, the match can be rescheduled during the times given.
+- Warm-ups are optional should a player request one. (Note: the warm up should not be from the mappool the players are due to play and must have a drain time of 4 mins or less.)
+- After warm ups (should there be any), both players will `!roll` to decide who bans/picks first. Loser of the roll bans first and winner of the roll picks first.
+- Each player is allowed one ban per match (Not Tiebreaker).
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:231]:
```suggestion
- Each player is allowed one ban per match. The tiebreaker cannot be banned.
```

File: wiki/Tournaments/CC/1/en.md [hunk 8]
```diff
@@ -8,11 +8,237 @@ tags:
...
+### Quarterfinals
+
+Sunday, 19 June 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Seraxis ![][flag_GB] | 3 | **4** | ![][flag_US] **zelda101** | [#1](https://osu.ppy.sh/community/matches/25737972) |
+| **Pelzio** ![][flag_GB] | **4** | 1 | ![][flag_US] MicMck101 | [#1](https://osu.ppy.sh/community/matches/25739013) |
+| **dong** ![][flag_GB] | **4** | 3 | ![][flag_GB] djsettle | [#1](https://osu.ppy.sh/community/matches/25741329) |
+| Hydria ![][flag_GB] | 2 | **4** | ![][flag_GB] rafalbrew | [#1](https://osu.ppy.sh/community/matches/25740307) |
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:205]:
```suggestion
| Hydria ![][flag_GB] | 2 | **4** | ![][flag_GB] **rafalbrew** | [#1](https://osu.ppy.sh/community/matches/25740307) |
```

File: wiki/Tournaments/CC/1/en.md [hunk 9]
```diff
@@ -8,11 +8,237 @@ tags:
...
+
+### Round of 16
+
+Saturday, 18 June 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Atrocious ![][flag_SE] | 3 | **4** | ![][flag_GB] **dong** | [#1](https://osu.ppy.sh/community/matches/25710268) |
+| Sopison ![][flag_GB] | 0 | **4** | ![][flag_GB] djsettle | [#1](https://osu.ppy.sh/community/matches/25711194) |
```
Comment (TPGPL) [inline wiki/Tournaments/CC/1/en.md:214]:
```suggestion
| Sopison ![][flag_GB] | 0 | **4** | ![][flag_GB] **djsettle** | [#1](https://osu.ppy.sh/community/matches/25711194) |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**